index.js

```diff
@@ -15,23 +15,19 @@ if (isSimpleWindowsTerm) {
 	ansiStyles.blue.open = '\u001b[94m';
 }
 
-var styles = (function () {
-	var ret = {};
```

Comment:
I don't think there's any particular reason to use an iffy here. Removing it to gain a bit (not too much) performance and make the code simpler.